authappliance/lib/ldap_proxy.py

```diff
@@ -0,0 +1,319 @@
+import socket
```

Comment:
Please add a file header according to other ``lib`` files.

Reply:
done!